### ISSUE TITLE
Map creators should be able to thumbs-down suggestions

### ISSUE THREAD

**Issue by Xinnoh** [ppy/osu-web#2178]
I often see modders wall modding a tonne of suggestions with really unhelpful ideas, and asking their friend to mass upvote them for kudosu, regardless of whether or not the mapper agreed with it. The mapper needs to be able to downvote discussions they did not find helpful. If there are suggestions that should be helpful but the mapper disagrees, other users can still outvote the mapper if it's deemed helpful by the community.
Thumbs-down would also really help other users know if a mapper accepted or rejected a suggestion, and would be a really helpful side-effect.

(imo) An alternate solution would be that the creator is the only person who can give kudosu, but others can still react with thumbs, which eliminates the problem.

**Comment by peppy** [comment]
It would be nice to show the mapper's response next to the vote options when the mapper does vote [mapper approves] [mapper dislikes] or something.

**Comment by Kibbleru** [comment]
Yeah, i think there could be some indication of whether the mapper actually took the modder's suggestion, or did not. Similar to how in modv1 people usually color coded what was fixed and what was rejected.
